lib/matplotlib/backends/backend_agg.py

```diff
@@ -568,10 +568,12 @@ def print_jpg(self, filename_or_obj, *args, **kwargs):
             background = Image.new('RGB', size, color)
             background.paste(image, image)
             options = restrict_dict(kwargs, ['quality', 'optimize',
-                                             'progressive'])
+                                             'progressive', 'dpi'])
 
             if 'quality' not in options:
                 options['quality'] = rcParams['savefig.jpeg_quality']
+            # Set the same dpi in both x and y directions
+            options['dpi'] = (options['dpi'], options['dpi'])
```

Comment:
If the dpi wasn't set from the options, should it fetch it from the rcparams? (or does that already happen in building kwargs)?